src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,11 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machines[] machines = new Machines[]{new Bulldozer(), new Excavator(), new Truck()};
+        for (Machines currentMachine : machines) {
+            System.out.println(currentMachine.doWork());
+            System.out.println(currentMachine.stopWork());
```

Comment:
The here you can simply call these methods:

```suggestion
            currentMachine.doWork();
            currentMachine.stopWork();
```